address-space-controller/src/main/java/io/enmasse/controller/auth/WildcardCertProvider.java

```diff
@@ -42,15 +37,15 @@ public Secret provideCert(AddressSpace addressSpace, String cn, Collection<Strin
                 String message = String.format("Requested 'wildcard' certificate provider but no secret '%s' found", wildcardSecretName);
                 throw new IllegalStateException(message);
             }
-            log.info("Copying wildcard certificate for {}", cn);
+            log.info("Copying wildcard certificate for {}", endpointInfo.getServiceName());
 
             Map<String, String> data = new LinkedHashMap<>();
             data.put("tls.key", wildcardSecret.getData().get("tls.key"));
             data.put("tls.crt", wildcardSecret.getData().get("tls.crt"));
 
```

Comment:
Is there any validation we ought to do on the provided certificate?  For instance ensuring the certificate SAN cover the range of hosts required by EndpointInfo?

Reply:
Yes and no. I think it would be nice, but it would be discovered by the user quickly that the service has not been configured with the correct certificate.

Reply:
Fine.